docker/spark/Dockerfile

```diff
@@ -24,6 +24,8 @@ ADD https://repo1.maven.org/maven2/ai/djl/onnxruntime/onnxruntime-engine/${DJL_V
 ADD https://repo1.maven.org/maven2/ai/djl/tensorflow/tensorflow-engine/${DJL_VERSION}/tensorflow-engine-${DJL_VERSION}.jar /usr/lib/spark/jars/djl-tensorflow-engine-${DJL_VERSION}.jar
 ADD https://repo1.maven.org/maven2/net/java/dev/jna/jna/${JNA_VERSION}/jna-${JNA_VERSION}.jar /usr/lib/spark/jars/
 RUN chmod -R +r /usr/lib/spark/jars/
+## TODO: Update url
+RUN pip3 install --no-cache-dir https://raw.githubusercontent.com/xyang16/djl/master/djl_spark-${DJL_VERSION}-py3-none-any.whl
```

Comment:
Can we build the wheel during dockder build, or build the wheel before docker build?

Reply:
To build the wheel during docker build, we need pull the djl repo. Is this OK.

Reply:
Build the wheel before docker build.